src/scss/settings/_variables.scss

```diff
@@ -40,9 +40,9 @@ $type: (
         default: (22, 28),
         narrow: (16, 20)
     ),
-    large--scaleUp: (
-        default: (32, 40),
-        narrow: (18, 24)
+    large--scaleDown: (
```

Comment:
Is this worth mentioning the in changelog?

Reply:
Yup updated.